### ISSUE TITLE
Developers Call #1

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/organization#26]
```meta
Date: 2021-03-04
Time: 4:00pm
Timezone: UTC
Duration: 1h
UTCTime: 2021-03-04 16:00 UTC
```

### Agenda

1. Syncing up on the [progress](https://github.com/orgs/btcpayserver/projects/10) of the new release **v1.0.7.0** and agreeing on the release date, announcement.
2. Looking at issues and PR's and moving the leftovers to [0.13 milestone](https://github.com/orgs/btcpayserver/projects/11)
3. Rockstardev Sunday's coding streams (Coding with Uncle Rockstar)
4. Questions from new contributors, PR's, PR's to be picked, etc.

### Check your timezone

https://everytimezone.com/s/9e37523e

### Join the call

LINK FOR CALL https://meet.jit.si/btcpayserverdevmeeting

### Calendar invite

Subscribe to the BTCPay Server calendar. More info [here](https://github.com/btcpayserver/organization#calendar).

**Comment by pavlenex** [comment]
Please note that the time/date for this call may be a subject to change, will update as soon as I have more information from people.

**Comment by Zaxounette** [comment]
Developer call N°1 notes:
Attendees: @dennisreimann , @Kukks , @pavlenex , @Zaxounette 

Discussing next release, 
v1.0.6.9 is scheduled to be released by approximately next Thursday, 11/03/2020.
This corresponds to milestone [0.12](https://github.com/orgs/btcpayserver/projects/10) in the BTCPay Server Projects.

Is confirmed four main subjects in this release:
* Greenfield Wallet API - [#2246](https://github.com/btcpayserver/btcpayserver/pull/2246)
* Wallet Setup Refactoring/Redesign - [#2164](https://github.com/btcpayserver/btcpayserver/pull/2164) / [#2296](https://github.com/btcpayserver/btcpayserver/pull/2296)
* Full BECH32 support - [#2181](https://github.com/btcpayserver/btcpayserver/pull/2181)
* Decouple payout logic [#2046](https://github.com/btcpayserver/btcpayserver/pull/2046)

This release should also include bumps in Spark server and Charge (c-lightning)

---- 

Regarding milestone [0.13](https://github.com/orgs/btcpayserver/projects/11) is discussed objectives to work on:

* Bootstrap 5 is in progress and should be available done by 0.13
* LN wallet UI refactoring: Will need more research, harder to do than on-chain wallet refactoring
* Webauth: Will be studied in concurrence with LNbank that can use this feature
* LNbank is discussed and it is determined that it should be first implemented as standalone and then adapted in concurrence with the plugin feature
* New foundation page for past supporters
* Discussing wallet page setup having duplicate actions with other pages and removing duplicates for clarity. Mainly duplicates between `wallet setup` and `wallet modify`

----
